python/ray/autoscaler/sdk.py

```diff
@@ -206,6 +207,32 @@ def request_resources(num_cpus: Optional[int] = None,
     return commands.request_resources(num_cpus, bundles)
 
 
+def configure_logging(log_style: Optional[str] = None,
+                      color_mode: Optional[str] = None,
+                      verbosity: Optional[int] = None):
+    """Configures logging for cluster command calls.
```

Comment:
cc @ericl I'm introducing this configuration function so that we don't get bombed by logging messages when using Tune + docker rsync